boa/src/vm/mod.rs

```diff
@@ -521,11 +509,17 @@ impl Context {
         const OPERAND_COLUMN_WIDTH: usize = COLUMN_WIDTH;
         const NUMBER_OF_COLUMNS: usize = 4;
 
+        let msg = if !self.vm.frame().exit_on_return {
+            " Call Frame "
+        } else {
+            " VM Start"
+        };
+
```

Comment:
Can you swap the order on this? I understand `Call Frame` is the more common one but I think having simpler logic trumps that.
```suggestion
        let msg = if self.vm.frame().exit_on_return {
            " VM Start"
        } else {
            " Call Frame "
        };

```